resources/js/components/DetailField.vue

```diff
@@ -1,16 +1,27 @@
 <template>
+
     <panel-item :field="field">
+
         <template slot="value">
+
             <div class="inline-flex items-center">
-        	    <div class="rounded mr-2" v-bind:style="{ backgroundColor: field.value, width: '20px', height: '20px' }"></div>
-        	    <div>{{ field.value }}</div>
-    	    </div>	
+
+                <div class="rounded mr-2" :style="{ backgroundColor: field.value, width: '20px', height: '20px' }"/>
+
+                <div>{{ field.value }}</div>
+
+            </div>
+
         </template>
+
     </panel-item>
+
 </template>
 
 <script>
+
 export default {
-    props: ['resource', 'resourceName', 'resourceId', 'field'],
+    props: [ 'resource', 'resourceName', 'resourceId', 'field' ]
 }
+
 </script>
```

Comment:
Why all the extra spacing between HTML? please remove.  The other code style updates are fine.